README.md

```diff
@@ -1,2 +1,23 @@
 # lobster-ice
 STEP 2020 Capstone Project :) 
+
```

Comment:
A step 0 is to prompt user to install and use node version 10 above (I tried 8 and below and they don't work, because there are some ES6 syntax in the service api node). They can do it via the node versions manager (nvm). A link to nvm's source repo is best (https://github.com/nvm-sh/nvm)

Reply:
okay thanks for catching this!